docs/clusterfuzzlite.md

```diff
@@ -7,19 +7,32 @@ permalink: /
 ---
 
 # ClusterFuzzLite
-ClusterFuzzLite is a lightweight, easy-to-use, fuzzing infrastructure that is
-based off [ClusterFuzz]. ClusterFuzzLite is designed to run as workflows on
-[continuous integration] (CI) systems, which means it is easy to set up and
-provides a familiar interface for users.
 
-Currently ClusterFuzzLite fully supports [GitHub Actions]. Extending support to
-other CI systems is in the works.
+ClusterFuzzLite makes [fuzzing] part of [Continuous Integration (CI)].
+ClusterFuzzLite is based on [ClusterFuzz].
+
+By fuzzing your code, ClusterFuzzLite can make it more secure as fuzzing is
```

Comment:
Small suggestion:

`Fuzzing is a highly effective technique for finding bugs in software. ClusterFuzzLite makes your code more secure by fuzzing changes before they even enter the codebase.`